examples/plot_plotting_metrics_with_error.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+============================
+Plotting Metrics with Errors
+============================
+"""
+# %%
+# Load and preprocess the data set
+# ================================
+# We download the data set using :meth:`fetch_openml` function in :class:`sklearn.datasets`.
+# The original Adult data set can be found at https://archive.ics.uci.edu/ml/datasets/Adult
+# There are some caveats to using this dataset, but we will use it solely as an example
+# to demonstrate the plotting metrics with error bars functionality
+#
+# We start by importing the various modules we're going to use:
+from fairlearn.experimental.enable_metric_frame_plotting import plot_metric_frame
+from fairlearn.metrics import MetricFrame
+from sklearn.datasets import fetch_openml
+from sklearn.metrics import recall_score, accuracy_score, confusion_matrix
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.model_selection import train_test_split
+import pandas as pd
+import numpy as np
+
+
+data = fetch_openml(data_id=1590, as_frame=True)
+X = data.data
+y = (data.target == '>50K') * 1
+X_train, X_test, y_train_true, y_test_true = train_test_split(
+    X, y, test_size=0.33, random_state=42)
+
+X_train_processed = pd.get_dummies(X_train)
+X_test_processed = pd.get_dummies(X_test)
```

Comment:
Small point, but doing this relies on `get_dummies()` being insensitive to data order, and also that the train and test sets both fully 'cover' the data. We really should have a pipeline being built here instead.